mix.exs

```diff
@@ -27,7 +27,8 @@ defmodule Scholar.MixProject do
       {:ex_doc, "~> 0.29", only: :dev, runtime: false},
       {:nx, "~> 0.5.1"},
       {:explorer, "~> 0.5.1", only: [:test, :dev]},
-      {:nimble_options, "~> 0.5.2 or ~> 1.0"}
+      {:nimble_options, "~> 0.5.2 or ~> 1.0"},
+      {:axon, "~> 0.5.1", override: true}
```

Comment:
No need to depend on Axon for now. Let's copy and paste log_softmax for now and we will figure out how to merge them later. :)